test/plugins/tokens.test.js

```diff
@@ -16,7 +16,7 @@ const getTokenMock = (token) => {
 
     mock.update = sinon.stub();
     mock.regenerate = sinon.stub();
-    mock.toJson = sinon.stub().returns(token);
+    mock.toJson = sinon.stub().returns(hoek.clone(token));
```

Comment:
The fact that the `GET` endpoint calls `toJson` and then `delete`s some of the fields was causing problems with other tests. It should have been like this all along.